### ISSUE TITLE
Cannot reproduce Kinetics accuracy, could anybody please help?

### ISSUE THREAD

**Issue by Kewenjing1020** [facebookresearch/SlowFast#255]
I tried to reproduce the experiments on Kinetics 400 (I only downloaded 214738 out of 246,535 videos).  I use the config configs/Kinetics/SLOWFAST_8x8_R50.yaml, the batch size, GPU num and other parameters are exactly the same. During training, I keep getting a warning "Failed to decode by pyav with exception: tuple index out of range". 
The accuracy I get is **"top1_err": 40.407398, "  top5_err": 19.408849**
Now I'm not sure what causes the accuracy gap.  Does the pyav warning or the incomplete dataset hurt the accuracy?

**Comment by takatosp1** [comment]
Hi @Kewenjing1020,

Thanks for playing with PySlowFast!
The results you shared is too low. I remember some other folks have provided some link to the data, could you try to download through the link they provided and try it again?
Thanks,

**Comment by BoPang1996** [comment]
I use the config configs/Kinetics/SLOWFAST_8x8_R50.yaml. The date I use is shared by Xiaolong Wang in Nonlocal which contains 234643 training videos and 19761 val videos. I do not have 16 nodes, so I trained the model on 2x8 V100 cards with 8 mini-batch on each card. The base learning rate is scaled to 0.2. The top-1 accuracy is 75.8, 1.2 lower than the official results.

For SLOWFAST_8x8_R101_101_101.yaml, the reproduced top-1 acc is 77.2, 0.7 lower than the official results.

For SLOW_8x8_R50.yaml, the reproduced top-1 acc is 74.0, 0.8 lower than the official results.

Does anyone else suffer from this problem? Is this caused by incomplete data?
